sdks/python/.python-version

```diff
@@ -0,0 +1 @@
+3.7.13
```

Comment:
this might be added accidentally?

Reply:
this is causing RAT failures

Reply:
Thanks, I've just removed it, should pass RAT now